.github/workflows/check-markdown.yml

```diff
@@ -0,0 +1,33 @@
+name: Check Markdown
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+permissions:
+  packages: read
+
+jobs:
+  lint-markdown:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3.8.1
+        with:
+          node-version: "18.x"
+          registry-url: "https://npm.pkg.github.com"
+          scope: "@paritytech"
+
+      - name: Install tooling
+        run: |
+            npm install -g markdownlint-cli
+            markdownlint --version
+
+      - name: Check Markdown
+        env:
+          CONFIG: .github/.markdownlint.yaml
+        run: |
+          markdownlint --config "$CONFIG" --ignore target .
```

Comment:
Does it not care about gitignore?

Reply:
I will test again but during my initial test, I got tons of errors originating from `./target` so I would say it does not.

Reply:
I checked again and I confirm, the `.gitignore` is not considered by default. 

We could  point the linter to the `.gitignore` file instead of ignoring only the target folder but I do not see any value of doing that, considering we only consider the .md files already. 

There are even patterns in `.gitignore` such as `polkadot.*` that may get in the way.